### ISSUE TITLE
Training data quality

### ISSUE THREAD

**Issue by lxystar00** [junyaoshi/ZeroMimic#4]
Thank you for the fantastic work!!

My question is about the quality of the training data. Given that the 3D trajectories extracted from web videos can't be directly compared against a ground truth, I was wondering what was your primary metric or method for determining that the trajectory quality was good enough to train a successful policy during development?
